package.json

```diff
@@ -33,5 +33,5 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "version": "0.9.27"
+  "version": "0.9.28"
```

Comment:
I suppose this library published alright? (I can't see a publishConfig section)

Reply:
I published it today, now at version 0.9.29 as 0.9.28 had been taken by recent changes to the lib made by Scott.